test/functional/cypress/specs/companies/add-company-spec.js

```diff
@@ -223,20 +223,59 @@ describe('Add company form', () => {
                   cy.get(selectors.companyAdd.backButton).should('be.visible')
                 })
 
-                it('should display "Add company" button', () => {
-                  cy.get(selectors.companyAdd.submitButton).should('be.visible')
+                it('should display "Continue" button', () => {
+                  cy.get(selectors.companyAdd.continueButton).should(
+                    'be.visible'
+                  )
                 })
 
-                context('when the "Add company" button is clicked', () => {
+                context('when the "Continue" button is clicked', () => {
                   before(() => {
+                    cy.get(selectors.companyAdd.continueButton).click()
+                  })
+
+                  it('should render the "Add company" page', () => {
+                    cy.get(selectors.companyAdd.title)
+                    cy.should('have.text', 'Add company').and(
+                      'have.prop',
+                      'tagName',
+                      'H1'
+                    )
+                  })
+
+                  it('should contain a form to add a sector', () => {
+                    cy.contains('DIT sector')
+                      .next()
+                      .get('select option:selected')
+                      .should('have.text', '-- Select DIT sector --')
+                      .parent()
+                      .parent()
+                      .parent()
+                      .next()
+                      .contains('Add company')
+                      .and('have.prop', 'tagName', 'BUTTON')
```

Comment:
Perhaps, we could consider using this syntax instead as it's shorter? (not sure if this works exactly the same)
```suggestion
                      .and('match', 'button')
```

Reply:
Thanks 👍